packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -664,12 +701,19 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 
 		if (!blobId) {
 			// We submitted this op while offline. The blob should have been uploaded by now.
-			assert(
-				pendingEntry?.status === PendingBlobStatus.OfflinePendingOp &&
-					!!pendingEntry?.storageId,
-				0x38d /* blob must be uploaded before resubmitting BlobAttach op */,
-			);
-			return this.sendBlobAttachOp(localId, pendingEntry.storageId);
+			if (
```

Comment:
this duplicates the assert condition? is it just leftover from debugging/testing?

Reply:
Thanks, it was leftover